kettle/buckets.yaml

```
@@ -15,3 +15,4 @@ gs://kubernetes-jenkins/logs/:
   exclude_jobs:
     - ci-test-infra-benchmark-demo
     - ci-kubernetes-coverage-unit
```

not super important but I think we should change these to be an exact match
```suggestion
    - ^ci-test-infra-benchmark-demo$
    - ^ci-kubernetes-coverage-unit$
```